components/03-sections/07-quickfacts/quickfacts.scss

```diff
@@ -3,147 +3,163 @@
+        background-repeat: no-repeat;
+        background-size: contain;
+        position: absolute;
+        right: 0;
+        top: 0;
+        bottom: 0;
+        z-index: -10;
+        opacity: 0.007;
+        width: 100%;
+        height: 100%;
+        overflow: hidden;
+    }
+
+    .row {
+        margin: 0px;
+    }
+
+    .quick-facts-content {
+        text-align: center;
+        z-index: 100;
+
+        h2 {
+            font-family: "kulturista-web", serif;
+            font-weight: 700;
+            margin-bottom: 20px;
+            font-size: 4.5rem;
+        }
+
+        p {
+            font-size: 1.2rem;
+            font-weight: 600;
+            width: 100%;
+            line-height: normal;
+        }
+    }
+
+    .department-heading {
+        margin-bottom: 30px;
+    }
+
+    h3 {
+        color: $blue;
+    }
+
+    .quick-facts-content h2 {
+        color: $orange-a11y;
+    }
+
+    .quick-facts-content p {
+        color: $blue;
+    }
+
+    &.blue-bg {
+        ::after {
+            opacity: 0.025;
+        }
+
+        h3 {
+            color: $white;
+        }
+
+        .quick-facts-content {
+            h2 {
+                color: $orange-a11y;
+            }
+
+            p {
+                color: $white;
+            }
+        }
+    }
+
+    &.grey-bg {
+        ::after {
+            opacity: 0.005;
+        }
+
+        h3 {
+            color: $blue;
+        }
+
+        .quick-facts-content {
+            h2 {
+                color: $orange-a11y;
+            }
+
+            p {
+                color: $blue;
+            }
+        }
+    }
+
+    &.orange-a11y-bg {
+        background-color: $orange-a11y;
+
+        h3 {
+            color: $white;
+        }
+
+        .quick-facts-content {
+            h2 {
+                color: $blue;
+            }
+
+            p {
+                color: $white;
+            }
+        }
+    }
+
+    @include media-breakpoint-down(lg) {
+
+        .quick-facts-content {
+            h2 {
+                font-size: 3.2rem;
+            }
+
+            p {
+                font-size: 1.0rem;
+            }
+        }
+
+    }
+
+    @include media-breakpoint-down(md) {
+        padding-top: 30px;
+        padding-bottom: 30px;
+
+        .quick-facts-content {
+            h2 {
+                font-size: 3.5rem;
+            }
+
+            p {
+                font-size: 1.1rem;
+            }
+        }
+    }
+	// For DLS ONLY ==> 5-Up Variant - removes last column from 6-up layout 
```

Comment:
I wouldn't add any styles because this will be compiled and could remove the sixth item.